src/platform/packages/private/kbn-reporting/common/types.ts

```diff
@@ -151,6 +152,7 @@ export interface ReportSource {
   created_at: string; // timestamp in UTC
   '@timestamp'?: string; // creation timestamp, only used for data streams compatibility
   status: JOB_STATUS;
+  scheduled_report_id?: string;
```

Comment:
Opened an Elasticsearch PR to add this field to the mapping: https://github.com/elastic/elasticsearch/pull/127827